src/apps/LoRaMac/common/LmHandler/packages/FragDecoder.c

```diff
@@ -336,6 +336,12 @@ int32_t FragDecoderProcess( uint16_t fragCounter, uint8_t *rawData )
 
         // Update the FragDecoder.FragNbMissingIndex with the loosing frame
         FragFindMissingFrags( fragCounter );
+
+        if( FragDecoder.Status.FragNbLost == 0 && fragCounter == FragDecoder.FragNb)

```

Comment:
There are missing parenthesis.
Could you please change to
```c
if( ( FragDecoder.Status.FragNbLost == 0 ) && ( fragCounter == FragDecoder.FragNb ) )
```

Reply:
Done.